website/source/docs/agent/configuration/client.html.md

```diff
@@ -90,6 +90,22 @@ client {
   receive work. This may be specified as an IP address or DNS, with or without
   the port. If the port is omitted, the default port of `4647` is used.
 
+- `server_join` `(map[string]string` - Specifies the list of server information
```

Comment:
I don't think map[string]string is the right type? It should be its own stanza similar to: https://www.nomadproject.io/docs/agent/configuration/acl.html